tests/CMakeLists.txt

```diff
@@ -48,3 +48,7 @@ gtest_discover_tests(Prim)
 add_executable(ZAlgo ./z.cc)
 target_link_libraries(ZAlgo gtest_main)
 gtest_discover_tests(ZAlgo)
+
+add_executable(PrimeSieve ./prime_sieve.cc)
+target_link_libraries(PrimeSieve gtest_main)
+gtest_discover_tests(PrimeSieve)
```

Comment:
- [ ] move `z.cc` to `tests/strings/`
- [ ] move `prime_sieve.cc` to `tests/math/numtheory/`

Reply:
follow `pvl/abridged` for where to put stuff